### PR TITLE
Fix spacing in examples

### PR DESCRIPTION
**Pull request by dsaxton**
Spacing in these examples looks a little weird in the docs: https://docs.prefect.io/core/concepts/schedules.html#clocks

### CODE REVIEW

**Comment by madkinsz** [comment]
Thanks for contributing! I think we should probably just run our standard `black` formatting on these examples, if you don't mind taking the time.

**Comment by dsaxton** [comment]
> Thanks for contributing! I think we should probably just run our standard `black` formatting on these examples, if you don't mind taking the time.

Agreed. Added in an f-string as well for good measure.

File: docs/core/concepts/schedules.md [hunk 1]
```diff
@@ -13,10 +13,12 @@ from prefect import task, Flow
 from datetime import timedelta
 from prefect.schedules import IntervalSchedule
 
+
```
[the remark this answers is not in the document]
Comment (tpdorsey) [reply docs/core/concepts/schedules.md:16]: Ah I see this is a PEP recommendation.

**Review by madkinsz — APPROVED** [review]
Thanks!

**Pull request merged**